jobs/locations_feature_engineering.py

```diff
@@ -15,9 +15,58 @@ def main(prepared_locations_source):
         "service_count", F.size(locations_df.services_offered)
     )
 
+    locations_df = explode_services(locations_df)
+
     return locations_df
 
 
+def explode_services(locations_df):
+    service_columns = services_lookup().items()
+    for column_name, service_description in service_columns:
+
```

Comment:
Can this be one line?
```suggestion
for column_name, service_description in services_lookup().items():
```
The name `service_columns` to me feels like it's only the columns.

Reply:
because I've moved the lookup to another file it's quite long on one line now so I've just moved the `.items()` method call to the line below I'm not introducing a different variable name here anymore.